SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentReader.cpp

```diff
@@ -23,6 +23,9 @@ DECLARE_CYCLE_STAT(TEXT("Reader ApplyFastArrayUpdate"), STAT_ReaderApplyFastArra
 DECLARE_CYCLE_STAT(TEXT("Reader ApplyProperty"), STAT_ReaderApplyProperty, STATGROUP_SpatialNet);
 DECLARE_CYCLE_STAT(TEXT("Reader ApplyArray"), STAT_ReaderApplyArray, STATGROUP_SpatialNet);
 
+const TArray<FName> SpatialGDK::ComponentReader::SpecialCaseProperties = { FName(TEXT("Role")), FName(TEXT("RemoteRole")),
```

Comment:
To get the special case properties to work, I have to check for both ReplicatedMovement and bRepPhysics. However, I think that everything inside ReplicatedMovement is getting replicated regardless of other conditions, as a "special case" and I'm pretty worried this is gonna break stuff. I may have to look at how to apply only one property out of a struct.